src/model/database/db_build.js

```diff
@@ -1,8 +1,7 @@
-const fs = require("fs");
-
+const { QueryFile } = require('pg-promise');
 const dbConnect = require("./db_connect.js");
 
-sql = fs.readFileSync(`${__dirname}/db_build.sql`).toString();
+const sql = file => QueryFile(path.join(__dirname, file), { minify: true });
```

Comment:
You are missing the `path` definition, i.e. inclusion of `const path = require('path')` at the top ;)

Reply:
Hey, wow thanks! Well-spotted!